esports_data_from_rimble/rimble/rimble_test.go

```diff
@@ -0,0 +1,343 @@
+package rimble_test
+
+import (
+	_ "embed"
+	"regexp"
+	"testing"
+
+	lom "github.com/samber/lo/mutable"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/blocky/attestation-service-examples/esports-data-from-rimble/rimble"
+)
+
+//go:embed testdata/match_data.json
+var matchDataJSON []byte
+
+// todo: add a bit to the README about this package and how to trigger tests and update the test data
```

Comment:
Please do not leave todos in the codebase.

Reply:
Thanks for catching